src/lib/swap/HtlcUtils.js

```diff
@@ -0,0 +1,168 @@
+/* global Nimiq */
+/* global BitcoinJS */
+/* global Errors */
+
+class HtlcUtils { // eslint-disable-line no-unused-vars
+    /**
+     * @param {unknown} data - Uint8Array
+     * @returns {NimHtlcContents}
+     */
+    static decodeNimHtlcData(data) {
```

Comment:
The web-offline build of CoreJS does not yet include the HTLC class (https://github.com/nimiq/core-js/pull/572), so we need to recreate its `dataToPlain()` method ourselves to be able to parse NIM HTLC data.